tests/claimWin.test.ts

```diff
@@ -7,107 +7,117 @@ import {expect} from "chai";
 import {program} from "./utils/constants";
 
 describe('Wins claiming', () => {
-   it('should not let a user claim twice', async () => {
```

Comment:
curious, diff looks weird, was a lot changed here?

Reply:
I'm just reformatted the file, I'll revert the formatting changes.

Reply:
Done